### PR TITLE
Cleaning code

### PR DESCRIPTION
**Pull request by marimeireles**

### CODE REVIEW

File: src/widget.ts [hunk 1]
```diff
@@ -167,7 +167,7 @@ export class CytoscapeView extends DOMWidgetView {
     this.model.on('change:cytoscape_layout', this._updateLayout, this);
     this.model.on('change:cytoscape_style', this._updateStyle, this);
     this.model.on('change:elements', this.value_changed, this);
-    this.model.on('change:pixel_ratio', this.value_changed, this);
+    this.model.on('change:@param ElementModel', this.value_changed, this);
```
Comment (ianhi) [inline src/widget.ts:170]: ?
This can't be right?

Comment (marimeireles) [reply src/widget.ts:170]: Oh my
It's not, thanks Ian!

File: src/widget.ts [hunk 2]
```diff
@@ -357,7 +354,6 @@ export class CytoscapeView extends DOMWidgetView {
 
   /**
    * Add the listeners for traits that are common to nodes and edges
-   * @param ElementModel
```
Comment (ianhi) [inline src/widget.ts:360]: looks like it somehow got moved from here

Comment (marimeireles) [reply src/widget.ts:360]: I was searching and misstyped ;p

**Pull request merged**